src/meltano/core/project_init_service.py

```diff
@@ -152,23 +149,15 @@ def echo_instructions(self):
         )
 
         click.echo("\nNext steps:")
-        click.secho("  cd ", nl=False)
-        click.secho(self.project_name, fg="magenta")
+
+        if not Path.cwd().samefile(self.project.root):
+            click.secho("  cd ", nl=False)
+            click.secho(self.project.root, fg="magenta")
```

Comment:
It should be possible to resolve this by attempting to make `self.project.root` relative to the current working directory using `pathlib`. If it can be made relative, print the relative path (which may just be the name e.g. `testingdir`), and if not, print `self.project.root`.